mapbox_vector_tile/decoder.py

```diff
@@ -109,8 +119,16 @@ def parse_geometry(self, geom, ftype):
                     dx = x
                     dy = y
 
-                    coords.append([x, 4096-y])
-
-        if ftype == POLYGON and len(coords) > 0:
-            coords.append(coords[0])
-        return coords
+                    coords.append([x, self.extents-y])
+
+        if ftype == POINT:
+            return coords
+        elif ftype == LINESTRING or ftype == POLYGON:
+            if parts:
+                if coords:
+                    parts.append(coords)
+                return parts[0] if len(parts) == 1 else parts
```

Comment:
Because the decoder just returns a list of coordinates (I just introduced a list of lists to denote parts in this pr), I was assuming that it was would be the job of the caller to interpret the points appropriately.

Maybe we should discuss the expectation of what the useful return types would be?